File: typescript/roman-numbers-reverse/src/Rule.ts
export class Rule {
    private _roman: string;
    private _arabic: number;

    public get roman(): string {
        return this._roman;
    }

    public get arabic(): number {
        return this._arabic;
    }

    public constructor(parameters: { roman: string, arabic: number }) {
        const {roman, arabic} = parameters;
        this._roman = roman;
        this._arabic = arabic;
    }
}

File: java/roman-numbers/src/main/java/Rule.java
public class Rule {
    private final int arabic;
    private final String roman;

    public Rule(int arabic, String roman) {
        this.arabic = arabic;
        this.roman = roman;
    }

    public int getArabic() {
        return arabic;
    }

    public String getRoman() {
        return roman;
    }
}
